### ISSUE TITLE
Deprecate PyPI openmm package

### ISSUE THREAD

**Issue by jchodera** [openmm/openmm#3131]
Long ago, @rmcgibbo created a [PyPI `openmm` package](https://pypi.org/project/openmm/) that simply distributes a useful Python CLI for an older version of OpenMM. @peastman is also a co-owner of this project.

We should deprecate or rename this project until we can identify a path to distributing OpenMM itself through PyPI.

For future reference in potentially building OpenMM for pip distribution, we should check out [`cibuildwheel`](https://github.com/pypa/cibuildwheel).

**Comment by jchodera** [comment]
For motivation, it looks like 129 people/month still make the mistake of trying to install `openmm` via `pip`:
https://pypistats.org/packages/openmm
![image](https://user-images.githubusercontent.com/3656088/120225605-88c96800-c213-11eb-8b58-45bfabddd422.png)

**Comment by rmcgibbo** [comment]
@peastman: feel free to remove me as a co-owner. It doesn't appear that I can remove myself.

**Comment by jchodera** [comment]
@peastman : Given #3211, perhaps we should see if there is a way to delete the `openmm` pip package?

**Comment by cclauss** [comment]
Whoever has rights to the PyPI record can delete the package but that leaves the space open for someone else to take over the name which might not be a good idea.  Instead it might be useful to see the PyPI message to point to the current install instructions.

**Comment by giadefa** [comment]
maybe just remove the files and add just a README pointing to the openmm.org
page for installation instructions

**Comment by peastman** [comment]
It looks like Robert added me as a maintainer.  Presumably that means I have the permissions needed to do that.  I've never built PyPI packages before, so I don't know much about what it involves.  Ideally if someone tries to `pip install openmm` it should give an error message telling them the correct way to install it and then not install anything.  Is there a way to do that?

**Comment by cclauss** [comment]
Yes.  I can help with that but I need a few days before I can focus on that.

**Comment by peastman** [comment]
Fantastic, thanks!

**Comment by mikemhenry** [comment]
I am happy to make `pip install openmm` install a package that says "this isn't the openmm you are looking for, install it via conda, pip distribution is in progress"

**Comment by swails** [comment]
Why not just get rid of that package on pypi and then push one when it's ready?

**Comment by mikemhenry** [comment]
If pypi lets you do that and keep the name squatted, than that would be even better IMHO.

**Comment by mattwthompson** [comment]
Name-squatting in general is not permitted and in the case of a deleted package it seems that the name is released. DI:

> Generally when a project is deleted, the name becomes available

https://discuss.python.org/t/are-the-names-for-deleted-projects-retained-on-pypi/29384/4

**Comment by Ragzouken** [comment]
The installation guide (https://docs.openmm.org/latest/userguide/application/01_getting_started.html#installing-openmm) gives pip as a supported way to install OpenMM. Does that mean this issue obsolete or is it still a bad idea to install with pip?

**Comment by mattwthompson** [comment]
It's this issue title that's out of date, actually (and I think can be closed). OpenMM packages were conda-only until last year (https://github.com/openmm/openmm/issues/3796), `pip install openmm` is fine.

**Comment by cclauss** [comment]
https://pypistats.org/packages/openmm 1,500+ daily downloads... Let's close this.
